CHANGELOG.md

```diff
@@ -5,63 +5,85 @@ CHANGELOG
-- Add prompt for availability zone in `pcluster configure` automated subnets creation.
-- Use different permissions in instance roles based on the scheduler and the node's role in the cluster.
-- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' state
-  in case recurrent failures are encountered when provisioning nodes.
-- Add tag `Name` to every shared storage with the value specified in the shared storage name config.
+- Add support for multiple subnets when using AWS Batch.
+- Allow full control on the name of CloudFormation stacks created by ParallelCluster by removing the `parallelcluster-`
+  prefix.
+- Add multiple queues and compute resources support for pcluster configure when the scheduler is Slurm.
+- Add prompt for availability zone in pcluster configure automated subnets creation.
+- Add configuration HeadNode.Imds.Secured to enable/disable restricted access to Instance Metadata Service (IMDS).
+- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' 
+  state in case recurrent failures are encountered when provisioning nodes.
 
 **CHANGES**
-- Drop support for SGE and Torque schedulers.
-- Drop support for CentOS8.
+- Remove support for SGE and Torque schedulers.
+- Remove support for CentOS8.
 - Change format and syntax of the configuration file to be used to create the cluster, from ini to YAML.
-- Use AWS Cloud Development Kit to define and generate CloudFormation templates. Add Node.js as requirement.
-- Rename all the commands, refactor parameters and use JSON as output format.
-- Remove interactivity from commands used to create, update, delete and describe a cluster.
+- Remove `--cluster-template`, `--extra-parameters` and `--tags` parameters for the `create` command.
+- Remove `--cluster-template`, `--extra-parameters`, `--reset-desired` and `--yes` parameters for the `update` command.
+- Remove `--config` parameter for `delete`, `status`, `start`, `stop`, `instances` and `list` commands.
+- Add `--suppress-validators`, `--validation-failure-level` and`--disable-update-check` parameters to `create` command.
+- Add `--suppress-validators` and `--validation-failure-level` parameters to `update` command.
 - Remove possibility to specify aliases for `ssh` command in the configuration file.
-- Distribute AWS Batch commands: `awsbhosts`, `awsbkill`, `awsbout`, `awsbqueues`, `awsbstat` and `awsbsub`
-  as a separate `aws-parallelcluster-awsbatch-cli` PyPI package.
+- Rename `createami` command to `build-image` command, remove `--ami-id`, `--os`, `--instance-type`,
+  `--ami-name-prefix`, `--custom-cookbook`, `--post-install`, `--no-public-ip`, `--cluster-template`, `--vpc-id`,
+  `--subnet-id`.
+- Add `--image-name`, `--config`, `--region` parameters to `build-image` command.
+- Add `delete-image` command with `--name`, `--region`, `--force` parameters.
```

Comment:
New commands / options / functionalities should be moved  in the enhancements section. 
Also the new build image requires a sentence:
  Renew custom AMI creation and management that leverages EC2 Image Builder